Formula/newrelic.rb

```diff
@@ -0,0 +1,14 @@
+class Newrelic < Formula
+  desc "Tool to interact with New Relic infrastructure"
+  homepage "https://https://github.com/newrelic/newrelic-cli"
+  url "https://github.com/newrelic/newrelic-cli/releases/download/v0.1.0/newrelic-cli-darwin-0.1.0.zip"
```

Comment:
As per https://docs.brew.sh/Acceptable-Formulae#we-dont-like-binary-formulae, this needs to be built from source.

Reply:
Thank you for the feedback.  I've updated the port to build from source.